src/components/modals/OperationDetails.js

```diff
@@ -357,6 +355,11 @@ const OperationDetails = connect(
                           value={fee}
                           alwaysShowSign={false}
                           subMagnitude={1}
+                          prefix={
+                            <Box mr={1} color="grey" style={{ lineHeight: 1.2 }}>
+                              {'≈'}
+                            </Box>
+                          }
```

Comment:
Couldn't find a better way to do this without rewriting a lot of code. If there's a better way please let me know :eyes:

Reply:
lgtm. i think we have the same concept in mobile and i was suprised it was not yet the case on desktop! https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/src/components/CurrencyUnitValue.js#L20-L21